src/pymmcore_widgets/_slider_dialog_widget.py

```diff
@@ -0,0 +1,54 @@
+import re
+from typing import Optional
+
+from pymmcore_plus import CMMCorePlus, PropertyType
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QDialog, QGridLayout, QLabel, QWidget
+
+from ._core import get_core_singleton, iter_dev_props
+from ._property_widget import PropertyWidget
+
+
+class SliderDialog(QDialog):
+    """A Dialog that shows range-based properties (such as light sources) as sliders.
+
+    Parameters
+    ----------
+    property_regex : str
+        a regex pattern to use to select property names to show.
```

Comment:
hard to tell without looking at the code what this parameter is supposed to be.  A simple example or description of what the regex needs to contain would be good